src/standard/configure.html

```diff
@@ -45,11 +45,13 @@
 
     // storage for configuration
     _setupConfigure: function(initialConfig) {
-      this._config = {};
+      // If we have more non-undefined properties, it is more efficient to clone whole object
+      // and then remove undesired properties
+      this._config = Object.create(initialConfig || {});
```

Comment:
Do you mean properties deletion is not needed? Because original comment below shows that filtering from `undefined` properties was intentional.